optimum/intel/openvino/modeling_base.py

```diff
@@ -264,6 +264,10 @@ def _from_transformers(
             "trust_remote_code": trust_remote_code,
         }
 
+        if model_id.startswith("timm/"):
+            model_kwargs["use_safetensors"] = False
+            model_kwargs["num_labels"] = config.num_classes
```

Comment:
Could you add a comment to explain that this comes from a mismatch between timm and transformers models ?

Reply:
Thanks !